### DISCUSSION TITLE
SDR++ server lag RPI3b+ & WIFI

### DISCUSSION THREAD

**Discussion by LukeA07** [AlexandreRouma/SDRPlusPlus#900] [General]
Hi All,

Love the software, it's been great listing to airband in my local area. Currently, I'm running an RPi3b+ with an RTL-SDR dongle (Soon to be hackrf). I run SDR++ server on it and it's connected to the network via WIFI. I then run SDR++ on my windows computer in my office to listen to it. Most of the time it runs quite well. The connection says it's streaming at about 10Mbit/s and I'm using int8 Sample type with compression. Every 40 seconds or so it'll seem like the stream is lagging quite a bit... The streaming speed drops to 0-2Mbit/s and the SDR++ client on the windows computer freezes until the stream rate increases again. I understand this isn't an ideal setup and that there are probably some packet loss issues with the wifi connection but it's the best I can do at the moment. Pretty new to SDR, RPIs, and networking so I was just wondering if there are any settings I can change to get better performance. 

Thanks all!

**Comment by AlexandreRouma** [comment]
WiFi is already bad enough, and the interface on the older rpis are even worse.
Just use a network connection or in if absolutely impossible, an external wifi dongle that's not complete trash.

**Reply by LukeA07** [reply]
Yeah I thought that might be the case, I'll have a bit of a look into it. Thanks heaps mate!
